sentry-android-core/src/main/java/io/sentry/android/core/AppStartState.java

```diff
@@ -0,0 +1,70 @@
+package io.sentry.android.core;
+
+import android.os.SystemClock;
+import java.util.Date;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.TestOnly;
+
+/** AppStartState holds the state of the App Start metric and appStartTime */
+final class AppStartState {
+
+  private static @NotNull AppStartState instance = new AppStartState();
+
+  /** appStart in milliseconds */
+  private @Nullable Long appStart;
+
+  /** appStartEnd in milliseconds */
+  private @Nullable Long appStartEnd;
```

Comment:
`l`: When adding `millis` to the variable name we can ditch the comment.
```suggestion
  private @Nullable Long appStartMillis;
  private @Nullable Long appStartEndMillis;
```